Formula/mockolo.rb

```diff
@@ -10,10 +10,32 @@ class Mockolo < Formula
     sha256 cellar: :any_skip_relocation, big_sur:       "cf1c3210d31de9d13ff6908f4a72596221b669b0c9e457f03df322130eaceaf5"
   end
 
-  depends_on xcode: ["12.5", :build]
+  if MacOS.version >= :big_sur
+    depends_on xcode: ["12.5", :build]
+  else
+    depends_on "swift" => :build
+  end
+
+  uses_from_macos "swift"
 
   def install
-    system "swift", "build", "-c", "release", "--disable-sandbox"
+    swift_f = Formula["swift"]
+
+    # Skip the shims for testing; do not merge
+    swift = if OS.mac? && MacOS.version <= :catalina
+      swift_f.opt_prefix/"Swift-#{swift_f.version}.xctoolchain/usr/bin/swift"
```

Comment:
If formulae are linking to Swift we may want to install the toolchain to `Swift-#{version.major_minor}.xctoolchain` instead so we don't have to keep rebuilding things on point releases.

Reply:
Yes, good point. I will adjust this.

Reply:
Alternatively we could install things into `libexec` so that the install name doesn't contain version information and just symlink to `*.xctoolchain` for use with Xcode.

Reply:
There's a few assumptions in the Swift build scripts which makes it hard to not be installing into a toolchain.